docs/data-sources/database_replica.md

```diff
@@ -0,0 +1,69 @@
+---
+page_title: "DigitalOcean: digitalocean_database_replica"
+---
+
+# digitalocean\_database\_replica
+
+Provides information on a DigitalOcean database replica.
+
+## Example Usage
+
+```hcl
+resource "digitalocean_database_cluster" "foobar" {
+        name       = "foobar"
+        engine     = "pg"
+        version    = "11"
+        size       = "db-s-1vcpu-1gb"
+        region     = "nyc1"
+        node_count = 1
+        tags       = ["production"]
+}
+
+# Create replica resource of cluster
+resource "digitalocean_database_replica" "read-01" {
+  cluster_id = digitalocean_database_cluster.foobar.id
+  name       = "terra-test-ro"
+  region     = "nyc3"
+  size       = "db-s-2vcpu-4gb"
+  tags       =  ["staging"]
+}
+
+# Create data source of replica
+data "digitalocean_database_replica" "my_db_replica" {
+  cluster_id = digitalocean_database_cluster.foobar.id
+  name       = "terra-test-ro"
+}
```

Comment:
Generally the use case for data sources is to access information about a resource that is not managed in Terraform. So rather than creating the cluster and replica here, a more real world example would probably just have have the data sources perhaps outputting some connection info. 

```suggestion
data "digitalocean_database_cluster" "example" {
  name = "example-cluster"
}

data "digitalocean_database_replica" "read-only" {
  cluster_id = data.digitalocean_database_cluster.example.id
  name       = "terra-test-ro"
}

output "replica_output" {
  value = data.digitalocean_database_replica.read-only.uri
}
```

Reply:
ah yes. that makes sense! thanks.